payment-account-age-witness.adoc

```diff
@@ -0,0 +1,196 @@
+== Payment account age witness
+:toc:
+
+....
+  Bisq-IP: 2
+  Layer: Exchange
+  Title: Payment account age witness
+  Author: Manfred Karrer <mk@nucleo.io>
+  Comments-Summary: No comments yet.
+  Comments-URI: Bisq Forum thread [1]
+  Status: Draft
+  Created: 2017-09-14
+  License: GPL Version 3
+....
+
+== Abstract
+
+This proposal describes a verification model for the age of the first trade with a certain payment account as protection against a fraud scheme in which a criminal has obtained illegal access of a foreign bank account and tries to buy Bitcoin with the stolen funds.
```

Comment:
bank accounts don't need to be foreign to be abused, I would drop 'foreign'

Reply:
Was meaning anyone else's but is obvious anyway, will remove "foreign".